packages/manager/modules/emailpro/src/emailpro.controller.js

```diff
@@ -7,6 +7,7 @@ export default /* @ngInject */ function EmailProCtrl(
   $q,
   $rootScope,
   $scope,
+  $state,
```

Comment:
Could you please confirm that it's needed for achieving a refactor?
Because we used to avoid dealing with `$state` from a Controller right?

Reply:
As I'm changing the tab system yes it is. But you're right I'll try to pass it through resolve